Getting-Started-with-Akka-http-signature.md

```diff
@@ -329,4 +329,19 @@ could not find actor for Actor[akka://rww/user/rootContainer/card]rww.ldp.LDPExc
         at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
         at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
 ```
+## Certificate Authority
+
+We have now shown how to use public and private keys to verify the identity of the user. But often we also want to let the user be able to confirm the identity of the server they are connecting to as well. This can be done through Certificate Authority. The user can view this on the rww-play server as it makes a CA for the locally hosted server.
+
+In order to test whether this functionality works, one first has to save rww-play's certificate locally by using the following command after the server has been run: 
```

Comment:
I would add a line to show what not doing this looks like. You get some form of error on connection. Show the error in detail.